tests/manual-test-cases/Group5-Functional-Tests/5-28-VICAdmin-Isolated.robot

```diff
@@ -0,0 +1,101 @@
+
+    Get Docker Params  ${output}  ${false}
+
+    Set Environment Variable  VIC-ADMIN  %{VCH-IP}:2378
+
+Teardown VCH With No WAN
+    Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+    Cleanup VIC Appliance On Test Server
+
+Login And Save Cookies
+    [Tags]  secret
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk %{VIC-ADMIN}/authentication -XPOST -F username=%{TEST_USERNAME} -F password=%{TEST_PASSWORD} -D /tmp/cookies-%{VCH-NAME}
+    Should Be Equal As Integers  ${rc}  0
+
+*** Test Cases ***
+Display HTML
+    Login And Save Cookies
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk %{VIC-ADMIN} -b /tmp/cookies-%{VCH-NAME}
+    Should contain  ${output}  <title>VIC: %{VCH-NAME}</title>
+
+WAN Status
+    Login And Save Cookies
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk %{VIC-ADMIN} -b /tmp/cookies-%{VCH-NAME}
+    Should contain  ${output}  <div class="sixty">Registry and Internet Connectivity<span class="error-message">
+
+Get Portlayer Log
+    Login And Save Cookies
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk %{VIC-ADMIN}/logs/port-layer.log -b /tmp/cookies-%{VCH-NAME}
+    Should contain  ${output}  Launching portlayer server
+
+Get VCH-Init Log
+    Login And Save Cookies
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk %{VIC-ADMIN}/logs/init.log -b /tmp/cookies-%{VCH-NAME}
+    Should contain  ${output}  reaping child processes
+
+Get Docker Personality Log
+    Login And Save Cookies
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk %{VIC-ADMIN}/logs/docker-personality.log -b /tmp/cookies-%{VCH-NAME}
+    Should contain  ${output}  docker personality
+
+Get Container Logs
+    Login And Save Cookies
+    ${rc}  ${output}=  Run And Return Rc and Output  docker %{VCH-PARAMS} pull ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${container}=  Run And Return Rc and Output  docker %{VCH-PARAMS} create ${busybox} /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${container}  Error
+    ${rc}  ${output}=  Run And Return Rc and Output  docker %{VCH-PARAMS} start ${container}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${vmName}=  Get VM Display Name  ${container}
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -sk %{VIC-ADMIN}/container-logs.tar.gz -b /tmp/cookies-%{VCH-NAME} | (cd /tmp; tar xvzf - ${vmName}/tether.debug ${vmName}/vmware.log)
+    Log  ${output}
+    ${rc}  ${output}=  Run And Return Rc and Output  ls -l /tmp/${vmName}/vmware.log
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc and Output  ls -l /tmp/${vmName}/tether.debug
```

Comment:
Can we also assert that these log files aren't empty?

Reply:
Sure, good idea.